class-gf-razorpay.php

```diff
@@ -0,0 +1,285 @@
+<?php
+
+require_once ('razorpay-sdk/Razorpay.php');
+use Razorpay\Api\Api;
+
+GFForms::include_payment_addon_framework();
+
+class GFRazorpay extends GFPaymentAddOn {
```

Comment:
I was wondering if we could enforce PSR-4 here instead. Might be much cleaner, seeing as it is a public repo. @riagarg5 let us follow our coding guide till we decide though.